Motion_planning/README.md

```diff
@@ -0,0 +1,31 @@
+# Resources for motion planning
+
+## Courses:
+### Beginners:
+
+1. https://www.edx.org/course/autonomous-mobile-robots
+2. https://www.coursera.org/learn/robotics-motion-planning
+
+### Intermediate:
+
+1. https://www.coursera.org/learn/motion-planning-self-driving-cars
+
+## Books:
+1. [Robot motion planning intro](http://ais.informatik.uni-freiburg.de/teaching/ss11/robotics/slides/18-robot-motion-planning.pdf)
+2. [Principles of robot motion](http://biorobotics.ri.cmu.edu/book/)
+3. [Planning Algorithms by S. M. Lavalle](http://planning.cs.uiuc.edu/booka4.pdf)
+4. [ Probabilistic robotics by S. Thrun](https://docs.ufpr.br/~danielsantos/ProbabilisticRobotics.pdf) 
+5. [Object oriented planning](https://www.diva-portal.org/smash/get/diva2:7803/FULLTEXT01.pdf)
+6. [Autonomous mobile robots by R. Siegwart, Nourbakhsh](http://home.deib.polimi.it/gini/robot/docs/siegwart.pdf)
+
+### Path planning Thesis:
+1. [KTH karl kurzer](https://www.researchgate.net/profile/Karl_Kurzer/publication/337719881_Path_Planning_in_Unstructured_Environments_A_Real-time_Hybrid_A_Implementation_for_Fast_and_Deterministic_Path_Generation_for_the_KTH_Research_Concept_Vehicle/links/5de6b10292851c83645fc109/Path-Planning-in-Unstructured-Environments-A-Real-time-Hybrid-A-Implementation-for-Fast-and-Deterministic-Path-Generation-for-the-KTH-Research-Concept-Vehicle.pdf)
+
+## Important libraries and webpages:
+1. [OMPL](https://ompl.kavrakilab.org/)
+2. [RRT](http://msl.cs.uiuc.edu/rrt/)
+3. [CMU motion planning course homepage](http://www.cs.cmu.edu/~./motionplanning/)
+
+## Important publications and research groups:
+1. http://ai.stanford.edu/~latombe/pub.htm
```

Comment:
please make in this format `[RRT](http://msl.cs.uiuc.edu/rrt/)`

Reply:
nope

Reply:
@patilunmesh please be a little serious and put some effort. This is your initiative after all. It should look good.